cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/StorageClient.java

```diff
@@ -47,4 +47,14 @@ public interface StorageClient<T> {
    * @return the root prefix for OpenHouse on the storage system
    */
   String getRootPrefix();
+
+  /**
+   * Checks if the path exists on the backend storage. Path is the absolute path to file including
+   * scheme. Scheme is not prefix for local and hdfs storage. See:
+   * https://github.com/linkedin/openhouse/issues/121
+   *
+   * @param path absolute path to a file including scheme
```

Comment:
So this comment is not always true because the scheme may not have been passed along for hdfs files?
In that case, may be clarify that the path may or not have scheme specified. If scheme is not specified then it is assumed to be hdfs. For all other storage types, scheme must be specified.

Reply:
Updated